### PR TITLE
Em 4599 fix overlapping comments section

### PR DESCRIPTION
**Pull request by Jefex7**
### JIRA link (if applicable) ###

[https://tools.hmcts.net/jira/browse/EM-4599](url)

### Change description ###

Added a margin to comment to fix overlapping comments section

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: projects/media-viewer/src/lib/annotations/annotations.module.ts
Comment (Jefex7) [inline projects/media-viewer/src/lib/annotations/annotations.module.ts:1]: I had an Issue where the provider was missing, I am not sure if this is the correct way to do it.

File: projects/media-viewer/src/lib/annotations/comment-set/comment-set-header/comment-set-header.component.ts [hunk 1]
```diff
@@ -1,29 +1,32 @@
-import { Component, EventEmitter, Input, OnDestroy, OnInit, Output, ViewEncapsulation } from '@angular/core';
+import { Component, EventEmitter, Inject, Input, OnDestroy, OnInit, Output, ViewEncapsulation } from '@angular/core';
```
Comment (MunishSharmaHMCTS) [inline projects/media-viewer/src/lib/annotations/comment-set/comment-set-header/comment-set-header.component.ts:1]: why do we have inject here

File: projects/media-viewer/src/lib/annotations/comment-set/comment/comment.service.ts
Comment (MunishSharmaHMCTS) [inline projects/media-viewer/src/lib/annotations/comment-set/comment/comment.service.ts:1]: might need unit test for these. Sonar cube won't pass

File: projects/media-viewer/src/lib/annotations/comment-set/comment-set-header/comment-set-header.component.ts (hunk 1 above)
Comment (Jefex7) [reply projects/media-viewer/src/lib/annotations/comment-set/comment-set-header/comment-set-header.component.ts:1]: That is a mistake I forgot to remove it.

**Review by MunishSharmaHMCTS — APPROVED** [review]

**Review by yogesh-hullatti — APPROVED** [review]

**Pull request merged**